test/testdata/dsl/rails/cattr_reader.rb

```diff
@@ -0,0 +1,25 @@
+# typed: strict
+
+class GoodUsages
+  extend T::Sig
+  cattr_reader :both, :foo
+  cattr_reader :no_instance, instance_accessor: false
+  cattr_reader :bar, :no_reader, instance_reader: false
```

Comment:
What is `:bar`? Why isn't it referenced elsewhere in the test?

Reply:
It's just there to make sure that things work properly when there are more than one symbol and an option is present.